### DISCUSSION TITLE
Moon brightness, is it adjustable?

### DISCUSSION THREAD

**Discussion by Keatah** [Stellarium/stellarium#3167] [Q&A]
Is there a way to adjust the brightness/magnitude of the Moon?

**Comment by gzotti** [comment]
No. We are adapting a few things, and every change needs tiny further adjustments, so it's possible that on some occasions something looks weird. Do you see such occasion?

**Comment by Keatah** [comment]
No visual anomalies. I was just used to older calculation methods I guess. This one doesn't seem to have enough contrast (for me).
